src/workers/pushNotificationHandler.js

```diff
@@ -165,9 +166,11 @@ export const messageHandler = async (message, isForeground) => {
     const { txId } = data;
 
     notifee.displayNotification({
+      // Unique ID for the notification, allowing you to update or remove it later
```

Comment:
That's cool, I didn't know we could update push notifications after they were displayed